noncompiled/view/ScalarDisplay.js

```diff
@@ -45,26 +45,41 @@ scalarDisplays.bedrock = new RealisticDisplay('bedrock');
 
 
 function ScalarDisplay(options) {
-	var color = options['color'] || 0x000000;
+	var minColor = options['minColor'] || 0x000000;
+	var maxColor = options['maxColor'] || 0xffffff;
 	var min = options['min'] || '0.';
 	var max = options['max'] || '1.';
 	var scalar = options['scalar'] || 'vScalar';
 	this.field = void 0;
 	this.scratch = void 0;
 	this.getField = options['getField'];
+	var minColor_str = '';
+	var maxColor_str = '';
```

Comment:
Hm, weird. I was originally thinking to implement `maxColor` as a shader uniform just like `this._color`, but I just tried doing that and after seeing what's involved I oddly think this is the way to go. 

The problem is `color` only gets used in one place, and that's in ScalarDisplay, but it doesn't make sense to pass a uniform to a generic shader just so it gets used by one particular Display. We should really just get rid of the `color` uniform. 

I do think the string conversion code needs to be a function. There should also be a conversion function for `max` and `min`, because its a PIMA when I get glsl errors whenever I write a new ScalarDisplay.

Reply:
I pushed a commit where I removed the color uniform